lib/ios-deploy.js

```diff
@@ -28,30 +30,37 @@ class IOSDeploy {
   }
 
   async install (app) {
+    const start = new Date();
     try {
       const bundlePathOnPhone = await this.pushAppBundle(app);
-      await this.installApplcation(bundlePathOnPhone);
+      await this.installApplication(bundlePathOnPhone);
+      log.info(`Installation is successful after ${new Date() - start}ms`);
```

Comment:
we already have a metrics tracking system, so you don't have to do this manually. see `BaseDriver::logEvent`. You could log an install start and install finish event.